plot_chunk_size_experiments.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import json
+import os
+
+if __name__ == "__main__":
```

Comment:
I would remove the scripts for the plots as the plots in the paper are generated with different scripts.